build.gradle.kts

```diff
@@ -107,7 +107,7 @@ dependencies {
     forge("net.minecraftforge:forge:1.8.9-11.15.1.2318-1.8.9")
     runtimeOnly("me.djtheredstoner:DevAuth-forge-legacy:1.2.1")
 
-    bundle("org.spongepowered:mixin:0.7.11-SNAPSHOT") {
+    bundle("org.spongepowered:mixin:0.8.7-SNAPSHOT") {
```

Comment:
Why did you change mixin version? Version 0.8.x contains some compatibility issues with older Minecraft versions. It causes some conflicts not only with the Minecraft version but also with other mods.